### PR TITLE
Added basic dependencies to Navigate,firebase and redux

### LINKED ISSUE #1: Install dependencies
## DESCRIPTION

Install all the libraries needed for the creation of the project.

- [x] Navigation
- [x] Redux-toolkit
- [x] Icons - vectorIcons
- [x] Firebase 
- [x] Formik
- [x] Optional - library to flip the flashcard

### PR DESCRIPTION
**Pull request by Raibrr**
# Description

## Screenshots
image

### CODE REVIEW

**Review by Eduardoespl — APPROVED** [review]
Good work, LGTM!

File: languageApp/android/app/google-services.json
Comment (BrandonVG) [inline languageApp/android/app/google-services.json:1]: You should hide this file or the file's content

**Review by BrandonVG — APPROVED** [review]
LGTM

File: languageApp/android/app/google-services.json
Comment (DerekAyala) [reply languageApp/android/app/google-services.json:1]: I agree.

**Review by DerekAyala — APPROVED** [review]
LGTM!

**Review by FebernalGober — APPROVED** [review]

**Review by Antonio3veces3 — APPROVED** [review]
Well done!

**Review by FranciscoSenestrari — APPROVED** [review]

**Pull request merged**